system-int-test/test/cmr/system_int_test/bootstrap/bulk_index/services_test.clj

```diff
@@ -93,3 +95,66 @@
          (is (= 3 (count refs)))))
      ;; Re-enable message publishing.
      (core/reenable-automatic-indexing))))
+
+(deftest bulk-index-service-revisions
```

Comment:
Note that we are not testing associations at all in this test. Does the variables bulk index test verify associations? From the code it looks like we are tracking associated collections within the variables index, so we should be verifying that in bulk update as well.